src/Kunstmaan/PagePartBundle/Helper/FormWidgets/PageTemplateWidget.php

```diff
@@ -27,84 +28,65 @@ class PageTemplateWidget extends FormWidget
 {
 
     /**
-     * @var AbstractPagePartAdminConfigurator
+     * @var EntityManagerInterface
      */
-    protected $pagePartAdminConfigurator;
-
-    /**
-     * @var EntityManager
-     */
-    protected $em;
+    private $em;
 
     /**
      * @var PagePartAdminFactory
      */
-    protected $pagePartAdminFactory;
-
-    /**
-     * @var PagePartAdmin
-     */
-    protected $pagePartAdmin;
+    private $pagePartAdminFactory;
 
     /**
      * @var PageInterface
      */
-    protected $page;
-
-    /**
-     * @var FormFactoryInterface
-     */
-    protected $formFactory;
+    private $page;
 
     /**
      * @var Request
      */
-    protected $request;
+    private $request;
 
     /**
-     * @var array
+     * @var PagePartWidget[]
      */
-    protected $widgets = array();
+    private $widgets = array();
 
     /**
-     * @var PageTemplate[]
+     * @var PageTemplateInterface[]
      */
-    protected $pageTemplates = array();
+    private $pageTemplates = array();
 
     /**
-     * @var AbstractPagePartAdminConfigurator[]
+     * @var PagePartAdminConfiguratorInterface[]
      */
-    protected $pagePartAdminConfigurations = array();
+    private $pagePartAdminConfigurations = array();
 
     /**
      * @var PageTemplateConfiguration
      */
     protected $pageTemplateConfiguration;
 
```

Comment:
I would keep the PHPDoc above the constructor since you did provide it everywhere else.